### ISSUE TITLE
Download PDF Feature Retiring

### ISSUE THREAD

**Issue by Hawkeye4040** [dotnet/docs#27037]
I really wish you would not be retiring this feature. I think it's great. If you feel a print to PDF feature is too difficult I'll gladly maintain it myself in my free time despite how busy I am. I mean it truly is that easy and if it is not then let's just be blunt and say it wasn't done right then.

Is there actually a good reason for retiring this feature besides it just being in the way or another thing to have to maintain or something like that? What am I missing here? Seems like a pretty simple, straightforward feature that was working just fine. I do hope this isn't a case of someone in charge saying I don't like this personally so let's just get rid of it.

**Comment by BillWagner** [comment]
ping @jffriend

**Comment by jffriend** [comment]
@Hawkeye4040 Thanks for the feedback. There are several issues related to why we are retiring Download PDF. They are covered in the [blog post that we just posted](https://docs.microsoft.com/en-us/teamblog/retiring-download-pdf). This feature isn't something that could be maintained by someone outside our team and I won't dispute that we designed the feature poorly. It was a feature from the early days of Docs and never got love. Users can still print as PDF individual articles via their operating system support for print to PDF.

**Comment by Hawkeye4040** [comment]
Thank you for the link I'll be sure to leave my feedback there too. In sure at first glance it doesn't sound like a big deal but when working collaboratively it's nice to grab a PDF copy and annotate it yourself. I wasn't even aware of any issues with it despite using it often but I suppose I'll just make something myself this weekend when I'm bored 🧐

**Comment by adegeo** [comment]
That blog post does provide a survey you can answer: https://ncv.microsoft.com/T2yvXtnkoq

**Comment by jffriend** [comment]
@Hawkeye4040 If you do make something, then please share with the community. I'd be happy to post something on the blog if you have a reusable solution. Just let me know.

**Comment by Hawkeye4040** [comment]
Yeah I wouldn't mind showing you what I use sometime when I get around to it.

**Comment by gewarren** [comment]
@Hawkeye4040 There are some printing layout suggestions here: https://github.com/MicrosoftDocs/feedback/issues/3694

**Comment by Hawkeye4040** [comment]
Thank you! This is very helpful for what I'm trying to accomplish here.

**Comment by Hawkeye4040** [comment]
Hmm, I see the point here about having to account for various Web UI Elements. We'd certainly want something flexible able to respond to multiple page templates, or updating templates. Failure to account for that would definitely make this a nightmare to maintain.

Let me scratch my head for a while because this is handy but certainly not handy enough to justify a whole work week or anything like that.
